### PR TITLE
Add flag to enable WRF-Chem module

### PR DESCRIPTION
**Pull request by jkrajniak**

### CODE REVIEW

**Review by tldahlgren — DISMISSED** [review]
LGTM. Defer merge decision until at least one maintainer has a chance to comment.

File: var/spack/repos/builtin/packages/wrf/package.py [hunk 1]
```diff
@@ -110,6 +110,11 @@ class Wrf(Package):
         default=True,
         description="Parallel IO support through Pnetcdf library",
     )
+    variant(
+        "chem"
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/wrf/package.py]:
```suggestion
        "chem",
```

**Review by tldahlgren — COMMENTED** [review]
Okay.  A syntax error needs to be fixed first:

```
spack\var\spack\repos\builtin\packages\wrf\package.py:115: [E999] SyntaxError: invalid syntax
```

**Review by tldahlgren — DISMISSED** [review]

**Comment by MichaelLaufer** [comment]
@jkrajniak Have you verified the correct functionality of WRF-Chem compiled with this?
Also,  it appears from the WRF-Chem documentation that this configure flag will only work for WRF v4.0+. Please verify that and add the appropriate compatibility conditions in the recipe. Thanks

**Comment by jkrajniak** [comment]
> @jkrajniak Have you verified the correct functionality of WRF-Chem compiled with this? Also, it appears from the WRF-Chem documentation that this configure flag will only work for WRF v4.0+. Please verify that and add the appropriate compatibility conditions in the recipe. Thanks

I've checked if the WRF-Chem is enabled during the compilation. But, right for 3.x this feature should be disabled. I've just started using WRF package, and spack package was super useful to install it.

**Review by MichaelLaufer — APPROVED** [review]
LGTM

**Review by tldahlgren — APPROVED** [review]

**Pull request merged**